cspell.json

```diff
@@ -0,0 +1,29 @@
+// This configuration file sets options for spell-checking pull requests
+// onto this repository. Please see
+//     https://github.com/streetsidesoftware/cspell/tree/main/packages/cspell#cspelljson
+// for more documentation about this config file.
+{
+    "ignoreWords": [
+        // Terms that come up in build / CI configuration.
+        "logissue", // Used in ##[vso.logissue] commands.
+        "pwsh",
+        "markdownlint",
+
+        // Words specific to quantum computing.
+        "eigenstate",
+        "qubit",
+        "qubits",
+        "qubitization",
+        "qubitized",
+
+        // Chemistry terminology
+        "nitrogenase",
+        "Hartree",
+        "Fock"
+    ],
+
+    "flagWords": [
+        // The Trotter–Suzuki decomposition isn't a verb.
+        "Trotterization"
```

Comment:
is this personal preference?

Reply:
Not per se, I don't think, so much as that examples of denominal verbs formed from proper nouns that refer to people are generally used to indicate "act in the manner of" rather than "use the invention of" (i.e.: ran across this neat paper that surveyed different semantic categories of denominal verbs formed from proper nouns: https://journals.openedition.org/lexis/4681#tocfrom2n8), such that it's much more unambiguous to explicitly state "use the invention of" (in this case, "use the Trotter–Suzuki decomposition").

There's also that the Trotter decomposition is a special case of the lowest-order Trotter–Suzuki decomposition. In most cases, we don't restrict to the lowest-order decomposition.